config/FCCAnalysisRun.py

```diff
@@ -476,6 +479,40 @@ def runLocal(rdfModule, fileList, output, batch):
     print  (" ")
     print  (" ")
 
+    if args.bench:
+        import json
+
+        analysis_path = sys.argv[1].removesuffix('/analysis_stage1.py')
```

Comment:
What if the python file is not called `analysis_stage1.py`?
Better:
```suggestion
        analysis_path = os.path.dirname(sys.argv[1])
```

Reply:
Yeah, its better to run benchmark on any of the stages